cypress/e2e/alerts.cy.js

```diff
@@ -1,25 +1,63 @@
 describe('Cypress application', () => {
-  before(() => {
-
+  const alertMessage = {
+    clickedAButton: 'You clicked a button',
+    allertAppearedAfter5Sec: 'This alert appeared after 5 seconds',
+    doYouConfirmAction: 'Do you confirm action?',
+    enterYourName: 'Please enter your name'
+  };
+
+  beforeEach(() => {
+    cy.visit('/');
   });
 
+  // eslint-disable-next-line max-len
   it('should have the ability to assert automatically resolved alerts', () => {
-
+    cy.get('#alertButton')
+      .click();
+    cy.on('window:alert', (alert) => {
+      expect(alert).to.equal(alertMessage.clickedAButton);
+    });
   });
 
   it('should have the ability to assert scheduled allert', () => {
-
+    cy.get('#timerAlertButton')
+      .click();
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(5000);
+    cy.on('window:alert', (alert) => {
+      expect(alert).to.equal(alertMessage.allertAppearedAfter5Sec);
+    });
   });
 
   it('should autimatically resolve alerts', () => {
-
+    cy.get('#confirmButton')
+      .click();
+    cy.on('window:allert', (alert) => {
```

Comment:
Typo in the event name 'window:allert', it should be 'window:alert'. Make sure to use the correct event names to avoid unexpected behavior.

Reply:
Thanks, fixed.